apps/scan/backend/src/scanners/custom/app_scan.test.ts

```diff
@@ -130,27 +139,19 @@ test('configure and scan hmpb', async () => {
   );
 });
 
-beforeEach(() => {
-  mockFeatureFlagger.enableFeatureFlag(
-    BooleanEnvironmentVariableName.SKIP_ELECTION_PACKAGE_AUTHENTICATION
-  );
-});
-
 test('configure and scan bmd ballot', async () => {
   await withApp(
     {},
     async ({ apiClient, mockScanner, mockUsbDrive, logger, mockAuth }) => {
       await configureApp(apiClient, mockAuth, mockUsbDrive, { testMode: true });
 
       mockScanner.getStatus.mockResolvedValue(ok(mocks.MOCK_READY_TO_SCAN));
-      await waitForStatus(apiClient, { state: 'ready_to_scan' });
```

Comment:
In general we never see `ready_to_scan` in tests anymore unless there's something preventing the paper from being scanned, like we need to sync CVRs or someone is logged in.